cmd/maya-apiserver/cstor-operator/controller.go

```diff
@@ -19,7 +19,7 @@ package spc
 import (
 	"fmt"
 	"github.com/golang/glog"
-	apis "github.com/openebs/maya/pkg/apis/openebs.io/v1alpha1"
+	apisv1alpha1 "github.com/openebs/maya/pkg/apis/openebs.io/v1alpha1"
```

Comment:
Why dont we have a `cmd/maya-apiserver/cstor-operator/v1alpha2` for this entire thing.
It will help in reviews that were missed earlier as well.

Currently, reviewers are limited to view the diff only.

Reply:
Will put it.